modules/oe_translation_poetry/src/Plugin/tmgmt/Translator/PoetryTranslator.php

```diff
@@ -567,6 +618,34 @@ protected function getTranslatedJobsByLanguage(ContentEntityInterface $entity):
     return $result ?? [];
   }
 
+  /**
+   * Get a list of Poetry jobs that are completed for a given entity.
+   *
+   * These are the jobs which have been translated by Poetry in a certain
+   * request.
+   *
+   * @param \Drupal\Core\Entity\ContentEntityInterface $entity
+   *   The entity to look jobs for.
+   * @param object $job_info
```

Comment:
I would prefer to have here either an integer (`$job_info->tjid`) or already the JobInterface $job object.

Like this the $job_info argument is not clear what should be.